packages/@lwc/state/src/types.ts

```diff
@@ -52,5 +52,6 @@ export type DefineState = <
     computed: MakeComputed,
     update: MakeUpdate,
     fromContext: MakeContextHook<ContextShape>,
+    setAtom: <T>(a: Signal<T>, newValue: T) => void,
```

Comment:
I'd like to rethink the `atom`/`setAtom` names at some point & see if we can come up with something that has more immediate / imperative connotations. Doesn't need to be in this PR, though.

Reply:
sure thing, let's  brainstorm on it!